sql/Procedures.sql

```diff
@@ -0,0 +1,127 @@
+DROP procedure IF EXISTS `edit_password`;
+DELIMITER $$
+CREATE DEFINER=`root`@`localhost` PROCEDURE `edit_password`( user_id int,  new_password varchar(512))
```

Comment:
I think we shouldn't have this `DEFINER` here, it will cause some trouble when we deploy the database. The role of the definer is to specify some access privileges, which, in my opinion, we don't need

Reply:
I agree :+1: . I will resolve conflicts locally and make these changes.

Reply:
Thank you!